{{cookiecutter.project_slug}}/Makefile

```diff
@@ -22,14 +22,14 @@ RESULTS_PATH_ENV?=$(PROJECT_PATH_ENV)/$(RESULTS_PATH)
 
 NEURO_CP=neuro cp --recursive --update --no-target-directory
 
-SETUP_NAME?=setup-{{cookiecutter.project_slug}}
-TRAINING_NAME?=training-{{cookiecutter.project_slug}}
-JUPYTER_NAME?=jupyter-{{cookiecutter.project_slug}}
-TENSORBOARD_NAME?=tensorboard-{{cookiecutter.project_slug}}
-FILEBROWSER_NAME?=filebrowser-{{cookiecutter.project_slug}}
+SETUP_NAME?=setup-{{cookiecutter.project_slug.replace('_', '-')}}
```

Comment:
let's move this value as postfix `{{cookiecutter.project_slug.replace('_', '-')}}`